sdk/table/azure-table/README.md

```diff
@@ -0,0 +1,354 @@
+    ```
+
+3. To use an [Azure Active Directory (AAD) token credential](https://docs.microsoft.com/azure/storage/common/storage-auth-aad),
+   provide an instance of the desired credential type obtained from the
+   [azure-identity](https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/identity/azure-identity#credentials) library.
+   For example, [DefaultAzureCredential](https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/identity/azure-identity#defaultazurecredential)
+   can be used to authenticate the client.
+   
+   This requires some initial setup:
+   * [Install azure-identity](https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/identity/azure-identity#install-the-package)
+   * [Register a new AAD application](https://docs.microsoft.com/azure/active-directory/develop/quickstart-register-app) and give permissions to access Azure Storage
+   * [Grant access](https://docs.microsoft.com/azure/storage/common/storage-auth-aad-rbac-portal) to Azure Table data with RBAC in the Azure Portal
+   * Set the values of the client ID, tenant ID, and client secret of the AAD application as environment variables: 
+     AZURE_TENANT_ID, AZURE_CLIENT_ID, AZURE_CLIENT_SECRET
+   
+   Use the returned token credential to authenticate the client:
+    ```python
+        from azure.identity import DefaultAzureCredential
+        from azure.table import TableServiceClient
+        token_credential = DefaultAzureCredential()
+    
+        queue_service_client = TableServiceClient(
+            account_url="https://<my_account_name>.table.core.windows.net",
+            credential=token_credential
+        )
+    ```
+
+#### Creating the client from a connection string
+Depending on your use case and authorization method, you may prefer to initialize a client instance with a storage
+connection string instead of providing the account URL and credential separately. To do this, pass the storage
+connection string to the client's `from_connection_string` class method:
+
+```python
+from azure.table import TableServiceClient
+
+connection_string = "DefaultEndpointsProtocol=https;AccountName=xxxx;AccountKey=xxxx;EndpointSuffix=core.windows.net"
+service = TableServiceClient.from_connection_string(conn_str=connection_string)
+```
+
+The connection string to your storage account can be found in the Azure Portal under the "Access Keys" section or by running the following CLI command:
+
+```bash
+az storage account show-connection-string -g MyResourceGroup -n MyStorageAccount
+```
+
+## Key concepts
+The following components make up the Azure Table Service:
+* The storage account itself
+* A table within the storage account, which contains a set of entities
+* An entity within a table, as a dictionary
+
+The Azure Table client library for Python allows you to interact with each of these components through the
+use of a dedicated client object.
```

Comment:
might want to re-word a little since it makes me think I should be able to get an EntityClient